app/src/org/commcare/android/util/FileUtil.java

```diff
@@ -395,4 +396,20 @@ public static String getDumpDirectory(Context c){
                 return null;
             }
         }
+
+    public static Properties loadProperties(Context c) throws IOException {
+        String[] fileList = { "local.properties" };
+        Properties prop = new Properties();
+        for (int i = fileList.length - 1; i >= 0; i--) {

```

Comment:
Is there a reason this is a for-loop even though there is only one file in the array of files?